java/feature-layer-update-attributes/README.md

```diff
@@ -1,4 +1,5 @@
-# Feature layer update attributes         
+# Feature layer update attributes
+### Category: Manage Data
 This sample demonstrates how to edit attributes of feature in a feature layer.      
 
 ![Feature Layer Update Attributes App](feature-layer-update-attributes.png)
```

Comment:
I know it's not caused by your changes, but the 'How it works' subtitle is incorrectly formatted for display.